src/main/scala/cr/pulsar/Publisher.scala

```diff
@@ -26,12 +26,36 @@ import org.apache.pulsar.client.api.{ MessageId, ProducerBuilder }
 import scala.concurrent.duration.FiniteDuration
 
 trait Publisher[F[_], E] {
+
+  /**
+    * It publishes a message in a synchronous fashion by using
+    * the given [[cats.effect.Blocker]].
+    */
   def publish(msg: E): F[MessageId]
```

Comment:
I think the interface shouldn't know about `Blocker`

Reply:
Yeah that was a comment that was on the implementation and I moved it to the interface... Will update.